.github/CODEOWNERS

```diff
@@ -189,7 +189,7 @@
 
 
 # Exploration project.
-/core/controllers/editor*.py @aks681
+/core/controllers/editor*.py @aks681 @rohitkatlaa
```

Comment:
Any specific reason for this assignment? Also, is this a temporrary change?

Reply:
I wanted to gain experience with code review and since I am currently a mentor of the project that deals with controllers so I thought I could become a codeowner for a few controller files.

Reply:
Currently, github allows PR merge if anyone of `@aks681 @rohitkatlaa` approves the PR, so you can review the PR but leave it for @aks681 to approve the PRs.